### PR TITLE
Theia extension for display the state of the different machines

### PR DESCRIPTION
**Pull request by olexii4**
Theia plugin for displaying information about Che workspace's machines (containers).
The name, the status, the servers list.

selection_118

Signed-off-by: Oleksii Orel <oorel@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,73 @@
...
+    cd browser-app
+    yarn watch
+
+Launch `Start Browser Backend` configuration from VS code.
+
+Open http://localhost:3000 in the browser.
+
+## Developing with the Electron example
+
+Start watching of the hello world extension.
```
Comment (AndrienkoAleksandr) [inline README.md]: Apply your extension name

File: browser-app/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "browser-app",
+  "version": "0.0.1",
+  "dependencies": {
+    "@theia/core": "latest",
```
Comment (AndrienkoAleksandr) [inline browser-app/package.json]: Set up, please, concrete version of dependencies 0.3.8 for example.

File: browser-app/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "@theia/editor": "latest",
+    "@theia/languages": "latest",
+    "@theia/markers": "latest",
+    "@theia/monaco": "latest",
+    "@theia/typescript": "latest",
+    "@theia/messages": "latest",
+    "theia-machines-extension": "0.0.1"
+  },
+  "devDependencies": {
+    "@theia/cli": "latest"
```
Comment (AndrienkoAleksandr) [inline browser-app/package.json]: Set up, please, concrete version of dependencies 0.3.8 for example.

File: electron-app/package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "scripts": {
+    "prepare": "theia build",    
+    "start": "theia start",
+    "watch": "theia build --watch"
+  },
+  "theia": {
+    "target": "electron"
+  }
+}
```
Comment (AndrienkoAleksandr) [inline electron-app/package.json]: Check ending files, please.

File: browser-app/package.json (hunk 3 above)
Comment (azatsarynnyy) [reply browser-app/package.json]: For now, only 0.3.7 is allowed.
I've just created [CQ](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16031) for 0.3.8

File: electron-app/package.json (hunk 4 above)
Comment (olexii4) [reply electron-app/package.json]: done

File: browser-app/package.json (hunk 3 above)
Comment (olexii4) [reply browser-app/package.json]: done

File: browser-app/package.json (hunk 2 above)
Comment (olexii4) [reply browser-app/package.json]: done

**Comment by olexii4** [comment]
1. After update version of dependencies to 0.3.8
selection_123
2. Without workspace runtime
selection_122

**Review by akurinnoy — APPROVED** [review]

**Review by evidolob — APPROVED** [review]

**Review by AndrienkoAleksandr — APPROVED** [review]

File: browser-app/package.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "browser-app",
+  "version": "0.0.1",
+  "dependencies": {
+    "@theia/core": "0.3.8",
```
Comment (azatsarynnyy) [inline browser-app/package.json:6]: @olexii4 FYI: CQ for 0.3.8 hasn't been approved yet https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16031

**Comment by olexii4** [comment]
selection_125

**Pull request merged**